ark/utils/metacluster_remap_gui/metaclustergui.py

```diff
@@ -0,0 +1,463 @@
+        self.im_m = self.ax_m.imshow(np.zeros((self.mcd.marker_count, self.mcd.metacluster_count)), norm=self.normalizer, cmap=self.heatmapcolors, aspect='auto', picker=True)  # noqa
+
+        self.ax_c.yaxis.set_tick_params(which='major', labelleft=False)
+        self.ax_c.set_yticks(np.arange(self.mcd.marker_count)+0.5)
+        self.ax_c.set_yticklabels(self.mcd.marker_names)
+        self.ax_c.set_xticks(np.arange(self.mcd.cluster_count)+0.5)
+        self.ax_m.set_xticks(np.arange(self.mcd.metacluster_count)+0.5)
+        self.ax_c.xaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+        self.ax_m.xaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+        self.ax_m.yaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+
+        # xaxis metacluster color labels
+        self.ax_cl.xaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+        self.ax_ml.xaxis.set_tick_params(which='both', bottom=True, labelbottom=True)
+        self.ax_cl.yaxis.set_tick_params(which='both', left=False, labelleft=True)
+        self.ax_cl.set_yticks([0.5])
+        self.ax_cl.set_yticklabels(["Metacluster"])
+        self.ax_ml.yaxis.set_tick_params(which='both', left=False, labelleft=False)
+
+        self.im_cl = self.ax_cl.imshow(np.zeros((1, self.mcd.cluster_count)), aspect='auto', picker=True, vmin=1, vmax=self.mcd.cluster_count)  # noqa
+        self.im_ml = self.ax_ml.imshow(np.zeros((1, self.mcd.metacluster_count)), aspect='auto', picker=True, vmin=1, vmax=self.mcd.cluster_count)  # noqa
+
+        # xaxis cluster selection labels
+        self.ax_cs.xaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+        self.ax_ms.xaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+        self.ax_cs.yaxis.set_tick_params(which='both', left=False, labelleft=True)
+        self.ax_cs.set_yticks([0.5])
+        self.ax_cs.set_yticklabels(["Selected"])
+        self.ax_ms.yaxis.set_tick_params(which='both', left=False, labelleft=False)
+        self.im_cs = self.ax_cs.imshow(np.zeros((1, self.mcd.marker_count)), cmap='Blues', aspect='auto', picker=True, vmin=-0.3, vmax=1)  # noqa
+        self.im_ms = self.ax_ms.imshow(np.zeros((1, self.mcd.marker_count)), cmap='Blues', aspect='auto', picker=True, vmin=-0.3, vmax=1)  # noqa
+
+        # xaxis pixelcount graphs
+        self.ax_cp.xaxis.set_tick_params(which='both', bottom=False, labelbottom=False)
+        self.ax_cp.yaxis.set_tick_params(which='both', left=False, labelleft=False)
+        self.ax_cp.set_ylabel("Pixels (k)", rotation=90)
+        self.ax_cp.set_xlim(0, self.mcd.cluster_count)
+        self.rects_cp = self.ax_cp.bar(
+            np.arange(self.mcd.cluster_count)+0.5,
+            np.zeros(self.mcd.cluster_count))
+        self.labels_cp = []
+        label_alignment_fudge = 0.08
+        for x in np.arange(self.mcd.cluster_count)+0.5+label_alignment_fudge:
```

Comment:
```suggestion
        for x in np.arange(self.mcd.cluster_count) + 0.5 + label_alignment_fudge:
```

Reply:
I believe our linter will complain that there should be a space in between each value connected by a mathematical operation.

Reply:
Not sure why, but pycodestyle doesn't find this case. I'll recheck manually for any others.